Classes/Form/AbstractMultiValueFormField.php

```diff
@@ -203,10 +248,139 @@ public function getRenderType()
 
     /**
      * @param string $renderType
-     * @return void
+     * @return MultiValueFieldInterface
      */
     public function setRenderType($renderType)
     {
         $this->renderType = $renderType;
+        return $this;
     }
+
+    /**
+     * @return boolean
+     */
+    public function getTranslateCsvItems()
+    {
+        return $this->translateCsvItems;
+    }
+
+    /**
+     * @param boolean $translateCsvItems
+     * @return MultiValueFieldInterface
+     */
+    public function setTranslateCsvItems($translateCsvItems)
+    {
+        $this->translateCsvItems = $translateCsvItems;
+        return $this;
+    }
+
+    /**
+     * @param array $items
+     * @return MultiValueFieldInterface
+     */
+    public function setItems($items)
+    {
+        $this->items = $items;
+        return $this;
+    }
+
+    /**
+     * @return array
+     */
+    public function getItems()
+    {
+        $items = [];
```

Comment:
Looks like the way to do that would be to move the adding of emptyOption to where $items is used when returned from the function. The problem is, we're building $items in many different ways and then finalising that built array with emptyOption in the end.

Although it would perhaps be cleaner, I think it's safer to keep adding of emptyOption in getItems so this method always returns the "final" array of items.